okrs_2019.md

```diff
@@ -0,0 +1,229 @@
+  - [ ] Implement 'Project Mode' ([#701])
+  - [ ] Make the SourceCred mirror module robust
+
+[#935]: https://github.com/sourcecred/sourcecred/issues/935
+[#907]: https://github.com/sourcecred/sourcecred/issues/907
+[#945]: https://github.com/sourcecred/sourcecred/issues/945
+[#701]: https://github.com/sourcecred/sourcecred/issues/701
+
+
+### Metrics
+
+We aren't sure what the right metrics are to track engagement and usage by
+downstream projects. For now, we'll measure "engagment", which is defined as:
+the project has taken a public action to use SourceCred, e.g. by linking to the
+cred attribution in their README, adding cred configuration files to their repository,
+or integrating a cred bot on GitHub. Actions that can happen without direct engagement
+from the project (e.g. listing their cred on Open Collective) don't count.
+
+For the purposes of the metrics below, "medium-sized projects" have at least 30 users
+in their cred attribution, and "large-sized projects" have at least 100.
+
+We'd like to see crypto projects start to use SourceCred as a way to transparently
+distribute tokens to contributors. For purposes of the metric, we're tracking public
+announcements or active distributions by well-known projects (i.e. no unknown ICOs).
+
+Finally, we're interested in seeing third-party integrations, of which the Open Collective
+integration is one.
+
+| Metric                                  | Goal | Q4 | Q1 | Q2 | Q3 | Q4 |
+| --------------------------------------- | ---- | -- | -- | -- | -- | -- |
+| # of engaged medium-sized projects      | 50   | 0  |    |    |    |    |
+| # of engaged larged-sized projects      | 5    | 0  |    |    |    |    |
+| # of announced token distributions      | 3    | 0  |    |    |    |    |
+| # of 3rd party integrations             | 3    | 0  |    |    |    |    |
+
+## SourceCred Helps Projects
+
+It's great if SourceCred has users. It's even better if we can show that we're
+actively helping our users. SourceCred has the potential to drive positive
+change in the communities using the technology, by fostering a greater sense of
+appreciation for contributors, and making contributing more fun and engaging.
+We'd like to be able to demonstrate that that SourceCred is helping projects in
+some of the following ways:
+
+- Engagement from new contributors
+- Project test coverage
+- Project documentation coverage
+- Speed with which issues get triaged
+
+### Initiatives
+
+Making SourceCred useful means finding a way to make the cred results salient
+to contributors to a project. Time-weighted cred (as described elsewhere in
+this document) will help a lot, since filtering cred to recent contributions
+will make the cred results more timely and relevant, especially to new
+contributors who won't quickly show up on the full-history analysis.
+
+We also may want to explore things like browser extensions or GitHub bots that
+surface cred information more readily.
```

Comment:
This Objective is a bit unclear to me at this point. 

Can you rephrase after this, perhaps you mean:

"Evaluate and document 5 potential initiatives to drive SC usage and implement 2 of them"